app/models/candidate.rb

```diff
@@ -27,16 +25,10 @@ def prepare(user)
       self.stil_id = user.profile.stil_id
     end
   end
-
-  def owner?(user)
-    user.present? && user.profile == profile
-  end
-
   def editable?
-    election.view_status == 2 || post.elected_by == 'Studierådet'
+    return (self.election.view_status == 2) || (self.post.elected_by == "Studierådet")
```

Comment:
Redundant `return` detected.<br>Redundant `self` detected.<br>Prefer single-quoted strings when you don't need string interpolation or special symbols.